language/entity-extraction/src/main/java/com/google/cloud/language/automl/entity/extraction/samples/ImportData.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.language.automl.entity.extraction.samples;
+
+// [START automl_natural_language_entity_import_data]
+
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.DatasetName;
+import com.google.cloud.automl.v1beta1.GcsSource;
+import com.google.cloud.automl.v1beta1.InputConfig;
+import com.google.protobuf.Empty;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+class ImportData {
+
+  // Import data from Google Cloud Storage into a dataset
+  static void importData(String projectId, String computeRegion, String datasetId, String path)
+      throws InterruptedException, ExecutionException, IOException {
+    // String projectId = "YOUR_PROJECT_ID";
+    // String computeRegion = "YOUR_COMPUTE_REGION";
+    // String datasetId = "YOUR_DATASET_ID";
+
+    // Instantiates a client
+    AutoMlClient client = AutoMlClient.create();
+
+    // Get the complete path of the dataset.
+    DatasetName datasetFullId = DatasetName.of(projectId, computeRegion, datasetId);
+
+    GcsSource.Builder gcsSource = GcsSource.newBuilder();
+
+    // Get multiple training data files to be imported from gcsSource.
+    String[] inputUris = path.split(",");
```

Comment:
Should the input just be an array of `gcsUris`?

Reply:
Also can this import whole directories or just individual files?

Reply:
This should be individual files. Will switch it out to an array.